sourcebundle/bundle.go

```diff
@@ -108,11 +110,17 @@ func OpenDir(baseDir string) (*Bundle, error) {
 			vs = make(map[versions.Version]sourceaddrs.RemoteSource)
 			ret.registryPackageSources[pkgAddr] = vs
 		}
+		deprecations := ret.registryPackageVersionDeprecations[pkgAddr]
+		if deprecations == nil {
+			deprecations = make(map[versions.Version]*RegistryVersionDeprecation)
```

Comment:
Unnecessary reallocation I think (see previous comment)

Reply:
Similar to the above, I'm getting nil pointer exceptions without it.

Reply:
I suppose that similar to the above we could just assign the nested map at the index of `pkgAddr` but I figured to was best to stick with the style of how `registryPackageSources` is handled above for consistencies sake.

```go
vs := ret.registryPackageSources[pkgAddr]
if vs == nil {
    vs = make(map[versions.Version]sourceaddrs.RemoteSource)
    ret.registryPackageSources[pkgAddr] = vs
}
```